packages/micropip/test_micropip.py

```diff
@@ -50,13 +50,14 @@ async def _mock_get_pypi_json(pkgname, **kwargs):
             json_data["releases"][version] = [
                 {
                     "filename": file,
-                    "url": "",
+                    "url": file,
                     "digests": {
                         "sha256": Wildcard(),
                     },
                 }
             ]
 
+        print("pkgname", pkgname, json_data)
```

Comment:
If this is added for debugging, please remove this before merging.

Reply:
Oops, thanks for catching it.